### ISSUE TITLE
A11y - Spatial Commerce - VoiceOver does not recognize "Open AR View" button

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5054]
When navigating the product page with VoiceOver, the 3D file and the "Open AR View" button are not recognized as interactive elements.  Furthermore, it does not provide appropriate feedback for its functionality.
According to the notes on the [A11y Circle Day Miro Board|https://miro.com/app/board/uXjVKro9LJs=/], it would be nice to create a button that is not only a *div* but also a *button*. 

!image-20241009-144145.png|width=1467,height=395,alt="image-20241009-144145.png"!

Steps to Reproduce:

# Enable VoiceOver or any screen reader.
# Navigate to the product page that includes the 3D model.
# Place the focus on the "Open AR View" button.
# Observe that the screen reader does not announce the button or describe its functionality.

h4. Expected Result:

The "Open AR View" element should be recognized and announced as a button with descriptive functionality like "Open AR view."

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38812**



[created from NEXT-38812, comment 502076]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-38812, comment 516309]
